app.rb

```diff
@@ -0,0 +1,177 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './rental'
+
+class App
+  def initialize
+    @persons = []
+    @books = []
+    @rentals = []
+  end
+
+  def run # rubocop:todo Metrics/CyclomaticComplexity
+    choose = select_options
+    choose = select_options while choose < 1 || choose > 7
+    case choose
+    when 1
+      books_list
+      puts 'first choose'
+    when 2
+      person_list
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      rentals_list
+    when 7
+      puts 'Bye'
+      exit
+    end
+  end
+
+  def select_options
+    puts
+    puts 'Please choose an option by entering a number: '
+    options = [
+      '1 - List all books',
+      '2 - List all people',
+      '3 - Create a person',
+      '4 - Create a book',
+      '5 - Create a rental',
+      '6 - List all rentals for a given person id',
+      '7 - Exit'
+    ]
+    puts options
+    gets.chomp.to_i
+  end
```

Comment:
- [ ] Your project works perfectly fine. However, according to the project requirements, these two methods are required to be in the main.rb and not in the app.rb file. Kindly make this change.

![image](https://user-images.githubusercontent.com/86318284/193070937-28432bcd-4481-4984-bd9a-e870f05c1422.png)